lightningd/channel_control.c

```diff
@@ -139,13 +139,21 @@ static void peer_got_announcement(struct channel *channel, const u8 *msg)
 		return;
 	}
 
+	/* To handle the case that Master was shut down before sends reply.
+	 * In this case, master may receive announcement more than once!
+	 */
+	tal_free(channel->remote_ann_node_sig);
+	tal_free(channel->remote_ann_bitcoin_sig);
```

Comment:
If the master was shut down we're dead as well. No need to handle that case here.

Reply:
Yes, I'll remove it.